docs/v0.5/runtimes/streaming.md

```diff
@@ -58,18 +45,16 @@ kubectl create namespace kafka
 helm install kafka --namespace kafka incubator/kafka --set replicas=1 --set zookeeper.replicaCount=1 --wait
 ```
 
-#### Create Kafka gateway with riff CLI
-
 The easiest way to create a KafkaGateway is using the riff CLI:
 
 ```sh
 riff streaming kafka-gateway create franz --bootstrap-servers kafka.kafka:9092
 ```
 
-You should see a deployment and service appear:
+You should see a deployment and service:
```

Comment:
or add `--tail` to the create command and know it came up correctly. While this currently holds true, it's not required for the test below to pass.